scalafmt-tests/src/test/resources/scala3/FewerBraces.stat

```diff
@@ -1802,3 +1802,312 @@ object a:
 object a:
    foo:
       erased bar => baz
+<<< rewrite to fewer braces: block simple
+rewrite.scala3.removeOptionalBraces = {
+  enabled = yes
+  fewerBracesMinStats = 2
+  fewerBracesMaxStats = 2
+}
+===
+foo
+  .mtd1 {
+    x + 1
+  }
+  .mtd2 {
+    x + 1
+    x + 2
+  }
+  .mtd3 {
+    x + 1
+    x + 2
+    x + 3
+  }
+>>>
+foo
+  .mtd1 {
+    x + 1
+  }
+  .mtd2:
+     x + 1
+     x + 2
+  .mtd3 {
+    x + 1
+    x + 2
+    x + 3
+  }
+<<< rewrite to fewer braces: func
+rewrite.scala3.removeOptionalBraces = {
+  enabled = yes
+  fewerBracesMinStats = 3
+  fewerBracesMaxStats = 3
+}
+===
+foo
+  .mtd1 { x =>
+    x + 1
+  }
+  .mtd2 { x =>
+    x + 1
+    x + 2
+  }
+  .mtd3 { x =>
+    x + 1
+    x + 2
+    x + 3
+  }
+>>>
+foo
+  .mtd1 { x =>
+    x + 1
+  }
+  .mtd2: x =>
+     x + 1
+     x + 2
+  .mtd3 { x =>
```

Comment:
I wouldn't count the parameter, I don't think it's very intuitive

Reply:
i think it counts the invisible block which contains the two statements, not the parameter. checking whether the block is invisible occasionally leads to non idempotent formatting as another rule could also remove the braces. but let me think if I can figure that out.